src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const formInputs = document.querySelectorAll('.form__input');
+const textArea = document.querySelector('.form__textarea');
```

Comment:
Maybe the better option would be to add the same class to textarea as inputs have